### PR TITLE
Created new class Person

### PR DESCRIPTION
**Pull request by anamariaburu**
Clasa noua Person

### CODE REVIEW

File: src/main/java/anab/Person.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+public class Person {
+
+  String FirstName;
```
Comment (smau-lavinia) [inline src/main/java/anab/Person.java]: please change the name of this variable to firstName

File: src/main/java/anab/Person.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    public String getFirstName() {
+        return FirstName;
+    }
+    public void setFirstName(String newFirstName) {
+        FirstName = newFirstName;
+    }
+
+
+    String LastName;
```
Comment (smau-lavinia) [inline src/main/java/anab/Person.java]: please change name to lastName

Comment (anamariaburu) [reply src/main/java/anab/Person.java]: Changes done in commit 419df3

File: src/main/java/anab/Person.java (hunk 1 above)
Comment (anamariaburu) [reply src/main/java/anab/Person.java]: Changes done under commit https://github.com/alexandruandriesei/svPreEmp/pull/57/commits/d4191ddaba8581bc639147c46207ab0eac419df3

File: src/main/java/anab/Person/Person.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package anab.Person;
+
+public class Person {
+
+    String firstName, lastName, Age, Gender, EmailAddress;
```
Comment (smau-lavinia) [inline src/main/java/anab/Person/Person.java]: please add private access modifier for the declared variables, based on encapsulation principle.

**Pull request merged**